configure.ac

```diff
@@ -3,6 +3,14 @@ AC_INIT([pick], [1.5.4], [pick-maintainers@calleerlandsson.com])
 AM_INIT_AUTOMAKE([subdir-objects])
 AC_CONFIG_HEADERS([config.h])
 AC_PROG_CC
+
+AC_CANONICAL_HOST
+build_linux=no
+case "${host_os}" in
+linux*) build_linux=yes ;;
```

Comment:
I haven't used `AC_CANONICAL_HOST` before and didn't find any documentation detailing the formatting of `host_os`. Is `linux*` a reliable pattern for catching all Linux-based OSs?